tibiawikisql/utils.py

```diff
@@ -297,7 +300,10 @@ def _parse_loot_entry(entry):
             key = subarg[0].strip()
             value = subarg[1].strip()
         entry[key] = value
-    return entry
+    if "item" in entry:
+        return entry
+    else:
+        return None
```

Comment:
This function was failing due to an improper table in TibiaWiki ([here](https://tibia.fandom.com/wiki/Loot_Statistics:Cublarc_the_Plunderer)). I've updated the table and the function should now be working.

IMO we could let the function chain crash when `item` does not exist - as it is a really unexpected behaviour from which we cannot recover from without losing data.

```suggestion
```

Reply:
Here is a screenshot of the improper table. It had a row with a nested random message log.

<img width="892" alt="image" src="https://user-images.githubusercontent.com/5060004/174857997-b1a52d76-7e62-4264-972a-c878fec0f812.png">

Reply:
Thanks for your comments, @antedeguemon! As I stated in [this comment](https://github.com/Galarzaa90/tibiawiki-sql/pull/43#discussion_r896083357) and the following one, I explained why I've decided to maintain this behavior - catch the exception, log the item that failed, continue parsing. I still feel it's better than killing the script.

Reply:
@gooshtavow sorry, I didn't see the previous thread. Seems excellent to me! 💯